lib/screens/login/login_repository.dart

```diff
@@ -2,8 +2,9 @@ import 'dart:convert';
 import 'dart:math';
 import 'package:ecellapp/core/res/errors.dart';
 import 'package:ecellapp/core/res/strings.dart';
+import 'package:ecellapp/core/utils/injection.dart';
+import 'package:ecellapp/core/utils/logger.dart';
 import 'package:http/http.dart' as http;
-import 'package:shared_preferences/shared_preferences.dart';
 
 abstract class LoginRepository {
   //Takes in login credentials (here id and password)
```

Comment:
Looks like you are attempting to write a **Documentation Comment**. Use `///` instead of `//` to do that.Triple slashes are used to write documentation comments (read [this](https://dart.dev/guides/language/effective-dart/documentation)). And always add a space after the slashes `// ` or `/// ` and then write the comment.
```suggestion
  /// Takes in login credentials (here id and password)
```